test_cronner.py

```diff
@@ -90,6 +91,56 @@ def fn():
         line = cronner.get_entries()
         self.assertEqual(line, 'template_var')
 
+    def test_custom_template(self):
+
+        template = '\n'.join(['concurrencyPolicy: Allow',
```

Comment:
Any reason you're going from list of strings to complete string to object here instead of just manually creating the object?

Reply:
For indentation formatting. If i did a multiline string i couldn't aline it correctly because the indentation would be included in the string itself.

Reply:
Can you just do something like:
```
{
    "concurrencyPolicy": "Allow", 
    "namespace": "test",
    ...
}
```?

Reply:
`serialize_kronjob` expects a string and passes it to `yaml.safe_load`